package.json

```diff
@@ -178,6 +179,7 @@
     "supertest": "^4.0.2",
     "ts-jest": "^26.4.1",
     "ts-node": "^9.1.1",
-    "typescript": "^4.0.3"
+    "typescript": "^4.0.3",
+    "uuid": "^3.4.0"
```

Comment:
Would you mind adding what this is used for to the commit message (and that we've added a package)?

Reply:
sure thing. 

just for reference here too. this was added to generate random UUIDs to make the factory data more realistic looking instead of just using the Fishery sequence for UUIDs.